### ISSUE TITLE
Check:imports command wrong reports when use trait in enums

### ISSUE THREAD

**Issue by alikhosravidev** [imanghafoori1/php_token_analyzer#18]
PHP: v8.1
Laravel: v9
microscope: v1.0.318
Os: Pop Os

`Autoload psr4:
"autoload": { "files": [ "app/helpers.php" ], "psr-4": { "App\\": "app/", "Database\\Factories\\": "database/factories/", "Database\\Seeders\\": "database/seeders/" } },`

```php
<?php

namespace App\Modules\LMS\Product\Enums;

use App\Core\General\Contracts\EnumHasPersianListTrait;

enum ProductAgeRange: string
{
    use EnumHasPersianListTrait;
}

```

Reported message:
1 Unused & wrong import:
use EnumHasPersianListTrait;
at app/Modules/LMS/Product/Enums/ProductAgeRange.php:9

**Comment by imanghafoori1** [comment]
This is fixed.
